src/app/utils/elementFactory.js

```diff
@@ -0,0 +1,23 @@
+const elementFactory = (tag, attributes, ...children) => {
+  const newElement = document.createElement(tag);
+
+  Object.entries(attributes).forEach(([key, value]) => {
+    if (key === 'className') {
+      newElement.classList.add(...value.split(' '));
+    } else {
```

Comment:
early return ⚔️

Reply:
Explained in private that is not possible in that case, because it needs to go into next foreach before return.